net/microsocks/Makefile

```diff
@@ -0,0 +1,44 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=microsocks
+PKG_VERSION:=1.0.3
+PKG_RELEASE:=1
+
+PKG_SOURCE:=microsocks-$(PKG_VERSION).tar.xz
+PKG_SOURCE_URL:=https://ftp.barfooze.de/pub/sabotage/tarballs/
```

Comment:
I wonder why not use tagged releases from their GitHub?
- https://github.com/rofl0r/microsocks